ansible/roles/notebooks/files/requirements.txt

```diff
@@ -14,3 +14,4 @@ plotly-geo==1.0
 psycopg2-binary==2.8.5
 yellowbrick==1.1
 widgetsnbextension
+fredapi==0.4.2
```

Comment:
Could you add this in alphabetical order? It will help us keep the dependency list a little more organized. Thanks!